libs/blocks/global-navigation/global-navigation.js

```diff
@@ -214,24 +214,24 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getUniversalNavLocale = (locale) => {
-  const LANGMAP = {
-    cs: ['cz'],
-    da: ['dk'],
-    de: ['at'],
-    en: ['africa', 'au', 'ca', 'ie', 'in', 'mt', 'ng', 'nz', 'sg', 'za'],
-    es: ['ar', 'cl', 'co', 'cr', 'ec', 'gt', 'la', 'mx', 'pe', 'pr'],
-    et: ['ee'],
-    ja: ['jp'],
-    ko: ['kr'],
-    nb: ['no'],
-    pt: ['br'],
-    sl: ['si'],
-    sv: ['se'],
-    uk: ['ua'],
-    zh: ['cn', 'tw'],
-  };
+export const LANGMAP = {
```

Comment:
I'd move the constant to the top of the file

Reply:
Done 👍 It was out of place between all those functions